libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -159,7 +193,7 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
     // @Description: Position controller P gain.  Converts the distance (in the latitude direction) to the target location into a desired speed which is then passed to the loiter latitude rate controller
     // @Range: 0.500 2.000
     // @User: Standard
-    AP_SUBGROUPINFO(_p_pos_xy, "_POSXY_", 5, AC_PosControl, AC_P),
+    AP_SUBGROUPINFO(_p_pos_xy, "_POSXY_", 5, AC_PosControl, AC_P_2D),
```

Comment:
Yes, I've tested it and the parameters are fine because they are in the same eeprom location and have the same type as before.  I've added a to-do to the testing list in the description but it's been tested once already.